.github/workflows/podman-test.yml

```
@@ -0,0 +1,207 @@
+          sleep 30
+
+          # Check container status
+          podman-compose -f podman-compose.ci.yml ps
+
+          # Check if container is healthy
+          container_id=$(podman ps -q --filter name=birdnet-go)
+          if [ -n "$container_id" ]; then
+            echo "✅ Container is running with ID: $container_id"
+            echo "Container logs:"
+            podman logs --tail 50 "$container_id"
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)